ert3/console/_console.py

```diff
@@ -13,6 +16,21 @@
 )
 
 
+def _build_init_argparser(subparsers):
+    init_parser = subparsers.add_parser("init", help="Initialize an ERT3 workspace")
+    pkg_examples_path = _get_ert3_examples_path()
+    ert_example_names = []
```

Comment:
This looks like a good candidate for a helper function in the sense of `_get_ert3_example_names`.